content/docs/dvclive/get-started.md

```diff
@@ -1,166 +1,196 @@
 # Get Started
 
 DVCLive is a Python library for logging machine learning parameters, metrics and
-other metadata in simple file formats, which is fully compatible with DVC.
+other metadata in simple file formats, which is fully compatible with DVC, the
+[VS Code Extension](https://marketplace.visualstudio.com/items?itemName=Iterative.dvc),
+and [Iterative Studio](https://studio.iterative.ai/).
 
-<admon type="tip">
+## Set up DVCLive
 
-If you use one of the supported [ML Frameworks](/doc/dvclive/ml-frameworks), you
-can jump directly to its corresponding page.
+First of all, you need to add DVCLive to your Python script:
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+<toggle>
+<tab title="Keras">
```

Comment:
Is there a criteria to the tabs we include here and their order? Just curious...

<img width="693" alt="image" src="https://user-images.githubusercontent.com/1477535/197716872-63187d5a-528e-49d1-a37f-611f21ff07fb.png">

E.g. rn the top 4 with most traffic are: Pytorch, HugginFace, Tensorflow, Keras (in that order).